src/components/ecosystems/Collections/CollectionsContainer.jsx

```diff
@@ -175,6 +175,11 @@ export default compose(
               activeItem['name'] = `${activeItem['name']}.md`
             }
 
+            if (items.find(item => item['name'] === activeItem['name'])) {
+              notifier.error('Item already exists with this name')
```

Comment:
@heislercreative let's remove this manual notifier line, since the rejected Error will show error message after the misc errors PR is merged

Reply:
Removed.